src/components/articles/Article.js

```diff
@@ -0,0 +1,78 @@
+import React, { Component } from 'react';
+import { Link } from 'react-router-dom';
+import { Card, Button } from 'react-bootstrap';
+import PropTypes from 'prop-types';
+
+class Article extends Component {
+  render() {
+    let articleImage;
+    const {
+      title,
+      description,
+      readtime,
+      created_at: createdAt,
+      author,
+      image,
+      slug,
+      body,
+    } = this.props;
+    if (!image) {
+      articleImage = 'https://www.mv-marchtrenk.at/files/images-sys/placeholder-news-full.png';
+    } else {
+      articleImage = image;
+    }
+    return (
+      <div className="flip-box">
+        <div className="flip-box-inner">
+          <div className="flip-box-front">
+            <Card>
+              <Card.Img variant="top" src={articleImage} className="homepage-card-image" />
+              <Card.Body>
+                <Card.Title>{title}</Card.Title>
+                <Card.Text>{description}</Card.Text>
+                <Card.Text>
+                  <span>By </span>
```

Comment:
Very nice implementation @Benkimeric . It is especially commendable that you were able to implement flipping cards. Also, very nice of you on implementing the feedback to include a space between 'By' and the user details.